editor/src/components/editor/actions/actions.ts

```diff
@@ -3486,21 +3486,19 @@ export const UPDATE_FNS = {
   ): EditorModel => {
     const existing = getContentsTreeFileFromString(editor.projectContents, action.filePath)
 
-    if (isTextFile(existing) && existing.fileContents.code === action.fileContents) {
```

Comment:
This check (introduced in https://github.com/concrete-utopia/utopia/pull/943) is no longer necessary, since we now only dispatch this action on changes that actually came from VS Code (and therefore we _know_ that the code changed)